src/query/frontend/ast/ast.hpp

```diff
@@ -3586,7 +3587,8 @@ class PatternComprehension : public memgraph::query::Expression {
   bool Accept(HierarchicalTreeVisitor &visitor) override {
     if (visitor.PreVisit(*this)) {
       if (variable_) {
-        variable_->Accept(visitor);
+        throw utils::NotYetImplemented("Variable in pattern comprehension.");
+        // variable_->Accept(visitor);
```

Comment:
Should this be removed now?

Reply:
I would prefer to keep it, because in later PRs I will remove throwing exception and will uncomment variable_->Accept.

Reply:
If it will be added in the next PR, feel free to keep it. If it will maybe be added in the next PR (depends on something else) I would remove it now so we don't need other PR to remove unnecessary comments. Feel free to resolve comment :muscle: